packages/amplify_flutter/lib/amplify.dart

```diff
@@ -182,10 +194,52 @@ class AmplifyClass extends PlatformInterface {
             underlyingException: e.toString());
       }
     }
-
     await DataStore.configure(configuration);
   }
 
+  Future<void> _parseConfigJson(String configuration) async {
+    try {
+      _config = _createConfigObject(configuration);
+      _configCompleter.complete(_config);
+    } catch (e) {
+      if (kDebugMode) {
```

Comment:
Prefer a utility function for this kind of stuff. Can add it to amplify_core, maybe.

Reply:
Writing a util for this, but keeping it in amplify_flutter since I don't expect it to be used by any other packages.